From e41ad77df94058788f2707b761850ef2380b1151 Mon Sep 17 00:00:00 2001
From: Arik Nemtsov <arik@wizery.com>
Date: Sat, 4 Jun 2011 12:25:09 +0300
Subject: [PATCH 69/75] wl12xx: support up to 16 SSIDs in a scheduled scan request to the FW

This is changed from a previously supported 8 SSIDs. Effective from
FW .61+
---
 drivers/net/wireless/wl12xx/scan.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/wireless/wl12xx/scan.h b/drivers/net/wireless/wl12xx/scan.h
index 8de3fea..3657a63 100644
--- a/drivers/net/wireless/wl12xx/scan.h
+++ b/drivers/net/wireless/wl12xx/scan.h
@@ -190,7 +190,7 @@ struct wl1271_cmd_sched_scan_config {
 } __packed;
 
 
-#define SCHED_SCAN_MAX_SSIDS 8
+#define SCHED_SCAN_MAX_SSIDS 16
 
 enum {
 	SCAN_SSID_TYPE_PUBLIC = 0,
-- 
1.7.0.4

